recipes/bash-completion/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "bash-completion" %}
+{% set version = "2.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/scop/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: fddd9e6739c972eea95d1f0e0e84fa9a17a5935ba21eab37308b27580cc4d790
+  patches:
+    # bug in upstream which will be fixed by release 2.10
+    - sysvdirs.patch
+
+    # add a conda specific library loaded flag
+    - track-conda-loaded.patch
+
+    # search for completions in the conda environment
+    - completions-path.patch
+
+    # adjust double load trap to use conda specific
+    - double-load-trap.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - autoconf
+    - automake
+
```

Comment:
This has a runtime dependency on bash

Reply:
So we don't allow users to install this and use it against their own bash? Or does it not work at all in that case?

Reply:
It is intended for use against the user's own bash, but it should also work with a conda installed version.  It only makes sense for interactive shell and was packaged as a hard dependency for the `conda-bash-completion` code (see https://github.com/conda-forge/staged-recipes/pull/10158).

Does this mean it has a `host` dependency on bash?  (Full disclosure: very new to the whole conda echo system).